lib/Dialect/TritonGPU/Transforms/RemoveLayoutConversions.cpp

```diff
@@ -29,6 +30,8 @@
 namespace mlir::triton::gpu {
 namespace {
 
+namespace tt = mlir::triton;
```

Comment:
needed?

Reply:
Maybe not worth it for single opcode...

Reply:
Oh, I see I have actually changed to use the full namespace in the code. Let me get rid of this

Reply:
ah I hadn't seen any use. Anyway that's a not :)